VogtObserver3/Customer.cs

```diff
@@ -8,16 +8,33 @@ class Customer : Observer
     {
         private readonly string _name;
         private readonly string _favoriteStore;
-        
+
+
         public Customer(string name, string favoriteStore)
         {
             _name = name;
             _favoriteStore = favoriteStore;
+           
 
         }
         public override void Update(float discount)
         {
-            
+            Console.WriteLine($"{_name} Discount is {discount}");
+        }
+
+        public Store AddCustomerToStore(Customer cust, Store store)
+        {
+            store.AddObserver(cust);
+
+            return store;
         }
+
+        public Store RemoveCustomerFromStore(Customer cust, Store store)
```

Comment:
p.s. no reason to return the store here.

This should be 

`public void RemoveCustomerFromStore(Customer cust, Store store)`

The reason being.. Store is a paramter. Whoever called your function already has access to the store (because they just passed it in as a parameter!). No reason to return it.

Reply:
Same goes for AddCustomerToStore